encoding.c

```diff
@@ -207,16 +205,16 @@ rb_enc_dummy_p(rb_encoding *enc)
     return ENC_DUMMY_P(enc) != 0;
 }
 
-static int enc_autoload(rb_encoding *);
+int rb_enc_autoload(rb_encoding *);
```

Comment:
This declaration seems unnecessary as it is duplicate with internal/encoding.h.

Reply:
Fixed. Thank you.